x-pack/plugins/actions/server/create_execute_function.test.ts

```diff
@@ -553,6 +737,53 @@ describe('execute()', () => {
 
     expect(mockedActionTypeRegistry.ensureActionTypeEnabled).not.toHaveBeenCalled();
   });
+
+  test('should skip ensure action type if action type is system action and license is valid', async () => {
+    const mockedActionTypeRegistry = actionTypeRegistryMock.create();
+    const executeFn = createExecutionEnqueuerFunction({
+      taskManager: mockTaskManager,
+      isESOCanEncrypt: true,
+      actionTypeRegistry: mockedActionTypeRegistry,
+      inMemoryConnectors: [
+        {
+          actionTypeId: '.cases',
+          config: {},
+          id: 'system-connector-.cases',
+          name: 'System action: .cases',
+          secrets: {},
+          isPreconfigured: false,
+          isDeprecated: false,
+          isSystemAction: true,
+        },
+      ],
+    });
+    mockedActionTypeRegistry.isActionExecutable.mockImplementation(() => true);
+    savedObjectsClient.get.mockResolvedValueOnce({
+      id: '123',
+      type: 'action',
+      attributes: {
+        actionTypeId: '.cases',
+      },
+      references: [],
+    });
+    savedObjectsClient.create.mockResolvedValueOnce({
+      id: '234',
+      type: 'action_task_params',
+      attributes: {},
+      references: [],
+    });
+
+    await executeFn(savedObjectsClient, {
+      id: 'system-connector-.case',
+      params: { baz: false },
+      spaceId: 'default',
+      executionId: 'system-connector-.caseabc',
+      apiKey: null,
+      source: asHttpRequestExecutionSource(request),
+    });
+
+    expect(mockedActionTypeRegistry.ensureActionTypeEnabled).not.toHaveBeenCalled();
```

Comment:
Hmmm ... this feels like we're not allowing system actions to be deny-listed (by not including them in the allow-list).  I guess I can see that, but ... not sure.